api.py

```diff
@@ -0,0 +1,58 @@
+import webapp2
+import json
+from google.appengine.api import urlfetch
+from google.appengine.ext import ndb
+
+class Issue(ndb.Model):
+  json = ndb.TextProperty()
+
+class Patch(ndb.Model):
```

Comment:
Looks like we don't need this, or did you still want to store the patches as descendants?